frontend/src/views/web/challenge/edit-challenge/edit-challenge-date.html

```diff
@@ -0,0 +1,45 @@
+<section class="ev-md-container text-center rm-overflow-y">
+    <div class="row">
+        <form name="editChallengeDateForm"
+              ng-submit="challenge.editChallengeDate(editChallengeDateForm.$valid)">
+            <div class="col s12">
+                <div class="input-field align-left">
+                    <span class="text-med-black w-400 fs-16">Start date and time</span>
+                    <input id="start_date" name="start_date" format="MM-DD-YYYY hh:mm a" locale="en"
+                           value="{{ challenge.page.start_date }}" ng-model="challenge.page.start_date"
+                           start-view="year" ng-model-options="{ updateOn: 'blur' }" placeholder="Start date"
+                           moment-picker="challenge.page.start_date"
+                           required>
```

Comment:
Please indent these lines here.

Reply:
I could not understand what you are exactly talking about. Should I indent line 9-12 deeper or shallower or do something else?

Reply:
Please `tab-out` 9-12 lines.

Reply:
I got it.